concourse/model/traits/publish.py

```diff
@@ -457,6 +525,16 @@ def oci_builder(self) -> OciBuilder:
     def use_buildkit(self) -> bool:
         return not self.raw['no-buildkit']
 
+    @property
+    def helmcharts(self) -> tuple[HelmchartCfg]:
+        return tuple([
```

Comment:
Why parsing to `list` prior to parsing to `tuple`?

Reply:
good point. changed to generator-expression